sixtyfps_runtime/interpreter/ffi.rs

```diff
@@ -595,8 +599,10 @@ impl Model for ModelAdaptorWrapper {
     }
 
     fn set_row_data(&self, row: usize, data: Value) {
-        let val: &ValueOpaque = unsafe { std::mem::transmute::<&Value, &ValueOpaque>(&data) };
-        self.0.set_row_data(row, val);
+        if row < self.row_count() {
```

Comment:
Why do we need this check?

Reply:
You are right, that is probably not needed.